chains/evm/cli/account/flagVars.go

```diff
@@ -1,9 +1,10 @@
 package account
```

Comment:
We have in `chains/evm/cli/deploy/deploy.go` also some private variables (bridgeAddress, relayerAddress, deployments) ehich you could also turn into public vars